File: java/ankit_kanyal/devnest_problems/phase_2/trees/KthSmallestInBst.java
// link- https://devsnest.in/algo-challenges/kth-smallest-element-in-bst?tab=question

static int solve(TreeNode root, int k){
   ArrayList<Integer> list=new ArrayList<>();
   helper(root,list);

   return list.get(k-1);
}
public static void helper(TreeNode root,ArrayList<Integer> list){
    if(root==null) return ;

    helper(root.left,list);
    list.add(root.val);
    helper(root.right,list);
}

File: Python/vedant_bhatnagar/DevsnestProblems_THA/Phase_2/Trees/kthsmallestinBST.py
def solve(root, k):
    # CODE HERE
    arr=[]
    def f(root):
        if root:
            f(root.left)
            arr.append(root.val)
            f(root.right)
    f(root)
    return arr[k-1]

####################################

i = 0
n = 0

def solve(root, k):
    
    global i,n
    if root:
        solve(root.left,k)
        i+=1
        if i==k:
            n = root.val
        solve(root.right, k)
    return n